src/app/interaction-model/OperationTypes.h

```diff
@@ -67,7 +67,7 @@ struct ReadState
 enum class WriteFlags : uint32_t
 {
     kTimed     = 0x0001, // Received as a 2nd command after a timed invoke
```

Comment:
Unaddressed issue here.

Reply:
I did not address because for a drop-in repleacement for ember-compatibility-functions we have https://github.com/project-chip/connectedhomeip/blob/master/src/app/util/ember-compatibility-functions.cpp#L764

It is also true that as we update the code we may choose to update some requirements (e.g. timed write can be decided by IM based on attribute flags). However the start is a drop-replacement.

Reply:
My point was that this is _not an invoke_ and _not a command_.  The documentation just makes no sense.

Reply:
I totally missed what you meant. Updated the comment.